stdlib/REPL/src/LineEdit.jl

```diff
@@ -1708,7 +1708,7 @@ end
 function getEntry(keymap::Dict{Char,Any},key::Union{String,Char})
     v = keymap
     for c in key
-        if !haskey(v,c)
+        if !(haskey(v,c)::Bool)
             return nothing
         end
         v = v[c]
```

Comment:
If `v[c]::Dict{Char,Any}`, an alternative may be to assert this here, similar to the suggestion in https://github.com/JuliaLang/julia/pull/46548#discussion_r958409744. I'm not familiar enough with this code to judge whether this holds, so I'm open to insights of more experienced developers.

Reply:
Thinking more about it, this would not work since `v` can be something else.